OpenRVS/classes/OpenLogger.uc

```diff
@@ -0,0 +1,16 @@
+class OpenLogger;
+
+var config bool bDebugLoggingEnabled;
+
+static function DebugLog(string s)
+{
+	if (bDebugLoggingEnabled)
+	{
+		log(s);
```

Comment:
Optional: in the log function, you can add a tag to it so that the log file prefaces your string with that tag.  EG
log("test",'OpenRVS');
would output to log file like this:
OpenRVS: test